Model/Processor/ImportProcessor.php

```diff
@@ -159,4 +171,30 @@ public function transformConfigToScopeConfig($path, array $config)
 
         return $return;
     }
+
+    /**
+     * Resolve the config value if it's an environment
+     * variable reference.
+     *
+     * @throws \UnexpectedValueException
+     *
+     * @param string $value
+     * @return string|false the original string, or the resolved
+     * string if it's a reference or false on error
+     */
+    public function resolveValue($value)
+    {
+        $value = preg_replace_callback(
+            '/\{env:([^:\}\{]+?)\}/',
+            function ($matches) {
+                $value = getenv($matches[1]);
+                if ($value === false) {
+                    throw new \UnexpectedValueException('Environment variable does not exist');
```

Comment:
The name of the missing environment variable should be given. Also variable substitution normally supports default values, which is normally pretty handy for distributed setups.